### PR TITLE
Toy Daggers!

### PR DESCRIPTION
**Pull request by ghost**
Part II of my meme cult toys
Toy Daggers that look like cult daggers! HONK!

:cl: ShiggyDiggyDo
add: You can now win Toy Daggers at your local arcade!
/:cl:

### CODE REVIEW

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -1075,6 +1076,22 @@
 	..()
 	to_chat(user, "<span class='info'>Station Time: [worldtime2text()]")
 
+/*
+ * Toy Dagger
+ */
+
+/obj/item/toy/toy_dagger
+	name = "toy dagger"
+	desc = "A cheap plastic replica of a dagger. Produced by THE ARM Toys, Inc."
```
Comment (DaedalusGame) [inline code/game/objects/items/toys.dm]: is this a player reference or does arm not play

**Comment by praisenarsie** [comment]
what about toy energy daggers

File: code/game/objects/items/toys.dm (hunk 1 above)
Comment (ShiggyDiggyDo) [reply code/game/objects/items/toys.dm]: It's a reference to Robustins cult end animation that players like to meme about, not any specific player

**Comment by ShiggyDiggyDo** [comment]
@praisenarsie I don't think we need toy energy daggers since we already have toy energy swords

**Comment by arsserpentarium** [comment]
give it to clown!

**Comment by ma44** [comment]
@ShiggyDiggyDo this is the cultist ritual kniff, not the energy pen/dagger.

**Comment by ShiggyDiggyDo** [comment]
It's supposed to be the cultist dagger, not the energy one.

File: code/game/objects/items/toys.dm (hunk 1 above)
Comment (AlexTheSysop42) [reply code/game/objects/items/toys.dm]: similar to how the nar-sour is made by THE ARM breweries

**Comment by AlexTheSysop42** [comment]
image

**Comment by ShiggyDiggyDo** [comment]
I don't know what I did but maybe I fixed it

**Comment by ma44** [comment]
All we need now are Ratvarian spears that bounce off of things that it hits. 

Bonus points if the player thinks they got impaled, but realized it just bounced off of them.

**Review by ShizCalev — APPROVED** [review]

**Pull request merged**